### PR TITLE
Feature/docker

### PR DESCRIPTION
**Pull request by bodya4243**

### CODE REVIEW

**Review by VolandevlodD — CHANGES_REQUESTED** [review]
LGTM, but please only show the changes related to Docker in this pull request. This pull request contains changes that are not related to this task!

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+# Final stage
+FROM openjdk:21-jdk-slim
+WORKDIR application
+COPY --from=builder application/dependencies/ ./
+COPY --from=builder application/spring-boot-loader/ ./
+COPY --from=builder application/snapshot-dependencies/ ./
+COPY --from=builder application/application/ ./
+ENTRYPOINT ["java", "org.springframework.boot.loader.launch.JarLauncher"]
+EXPOSE 8080
```
Comment (yevheniisynytsia) [inline Dockerfile]: please add empty line

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      - "${SPRING_LOCAL_PORT}:${SPRING_DOCKER_PORT}"
+      - "${DEBUG_PORT}:${DEBUG_PORT}"
+    environment:
+      SPRING_APPLICATION_JSON: '{
+      "spring.datasource.url": "jdbc:mysql://mysqldb:${DB_DOCKER_PORT}/${DB_DATABASE}?serverTimezone=UTC",
+      "spring.datasource.username": "${DB_USER}", 
+      "spring.datasource.password": "${DB_PASSWORD}", 
+      "jwt.secret": "${JWT_SECRET}"
+      }'
+      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```
Comment (yevheniisynytsia) [inline docker-compose.yml]: please add empty line

**Review by yevheniisynytsia — CHANGES_REQUESTED** [review]

**Review by yevheniisynytsia — CHANGES_REQUESTED** [review]
please update the branch to have files only related to docker task

**Comment by bodya4243** [comment]
So I should change comits into order. But now it's into master and the master is already merged from the order.

**Review by andrii-hoienko — APPROVED** [review]
Good job!

**Pull request merged**